app/test_split_integer.py

```diff
@@ -2,20 +2,25 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    test = split_integer(16, 4)
+    assert sum(test) == 16
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    test = split_integer(6,2)
```

Comment:
Consider removing the `test` variable and directly asserting the result of `split_integer(6, 2)` as per the checklist recommendation.